packages/location-field/src/index.js

```diff
@@ -430,7 +430,9 @@ class LocationField extends Component {
         .filter(feature => feature.properties.source !== "transit")
         .slice(0, suggestionCount);
 
-      const nonOtherFeaturesPresent =
+      // If no categories of features are returned, this variable is used to
+      // avoid displaying headers
+      const nonStandardFeaturesPresent =
```

Comment:
I thought stops and stations fall under "standard" features?

Reply:
Rename to `standardFeatures`, I think.

Reply:
I don't think we have any other features, I read it as "other" is the standard feature and the stations and stops were special. I've renamed to `transitFeatures` in 8ea983c6ab004d320de2da88e76bc66ee2be95e3 which I think is more clear.